files/events.yaml

```diff
@@ -1,4 +1,37 @@
 ---
+- title: "Richard Stallman - Copyright vs Community in the Age of Computer Networks"
+  date: 2014-11-07
+  time: '18:00'
+  speaker: Dr. Richard Stallman
+  description: |
+      "Location: Portland State University Hoffman Hall, 1833 SW 11th Avenue, Portland, Oregon
+
+      Copyright developed in the age of the printing press, and was designed
+      to fit with the system of centralized copying imposed by the printing
+      press.  But the copyright system does not fit well with computer
+      networks, and only draconian punishments can enforce it.
+
+      The global corporations that profit from copyright are lobbying
+      for draconian punishments, and to increase their copyright powers,
+      while suppressing public access to technology.  But if we
+      seriously hope to serve the only legitimate purpose of
+      copyright--to promote progress, for the benefit of the
+      public--then we must make changes in the other direction.
+
+     Speaker Bio:
```

Comment:
I put a pull request on the branch a while ago to fix that.

On Fri, May 15, 2015 at 5:37 PM, Cameron Brandon White <
notifications@github.com> wrote:

> In files/events.yaml
> <https://github.com/pdxacm/acm.pdx.edu/pull/60#discussion_r30456147>:
>
> > +  description: |
> > +      "Location: Portland State University Hoffman Hall, 1833 SW 11th Avenue, Portland, Oregon
> > +
> > +      Copyright developed in the age of the printing press, and was designed
> > +      to fit with the system of centralized copying imposed by the printing
> > +      press.  But the copyright system does not fit well with computer
> > +      networks, and only draconian punishments can enforce it.
> > +
> > +      The global corporations that profit from copyright are lobbying
> > +      for draconian punishments, and to increase their copyright powers,
> > +      while suppressing public access to technology.  But if we
> > +      seriously hope to serve the only legitimate purpose of
> > +      copyright--to promote progress, for the benefit of the
> > +      public--then we must make changes in the other direction.
> > +
> > +     Speaker Bio:
>
> ERROR: files/events.yaml:21:6 [/] document end expected (maybe invalid tab
> char found).
> testPyYaml
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pdxacm/acm.pdx.edu/pull/60/files#r30456147>.
>